.github/workflows/ci.yaml

```diff
@@ -139,6 +142,73 @@ jobs:
         if: ${{ always() }}
         run: rm -f cosign.key
 
+  create-and-upload-sbom:
+    name: create-and-upload-sbom
+    needs: [release]
+    if: startsWith(github.ref, 'refs/tags/')
+    runs-on: ubuntu-20.04
+
+    strategy:
+      matrix:
+        repo: ${{ fromJSON(needs.release.outputs.container_repos) }}
+
+    steps:
+      - name: Set up Go
+        uses: actions/setup-go@v2.1.5
+        with:
+          go-version: 1.17
+
+      - name: Install cosign
+        uses: sigstore/cosign-installer@v1.4.1
+        with:
+          cosign-release: 'v1.4.1'
+
+      - name: Install Syft
+        uses: anchore/sbom-action/download-syft@v0.6.2-draft
+
+      - name: Checkout
+        uses: actions/checkout@v2.4.0
+        with:
+          fetch-depth: 0
+
+      - name: Login to Container registries
+        run: |
+          echo "${{ secrets.DOCKER_PASSWORD }}" | docker login -u philipssoftware --password-stdin
```

Comment:
I think we have a matrix per registry, so we should only login to the registry for this particular matrix.

As we are then only logging in on one registry at a time we might reuse the docker login action.